styles/styles.css

```diff
@@ -55,22 +55,40 @@
   }
 }
 
+* {
+  box-sizing: border-box;
+}
+
 body {
-  font-size: var(--body-font-size-m);
-  margin: 0;
-  font-family: var(--body-font-family);
-  line-height: 1.6;
-  color: var(--text-color);
-  background-color: var(--background-color);
+  font-family: "HCo Decimal", Lato, "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  line-height: 1.4;
+  color: #36424b;
+  background-color: #fff;
   display: none;
+  margin: 0;
 }
 
 body.appear {
   display: unset;
 }
 
+body.no-body-scroll {
+  overflow: hidden;
+}
+
 header {
-  height: var(--nav-height);
+  background: #00C0E8;
+  height: 135px;
+}
+
+html:has(meta[name="template"][content="book"]) header {
```

Comment:
we should also avoid `:has` for low browser support: https://caniuse.com/css-has

Reply:
Should land this year in Firefox but you're right we should avoid it for now.